tests/scenarios/v2-addon-dev-typescript-test.ts

```diff
@@ -167,6 +174,18 @@ appScenarios
             `,
           },
         },
+        initializers: {
+          'demo.js': `
+            export function initialize() {
```

Comment:
it's been a while since I've used initializers, but why is this export needed? (since it's included in the default export below)

Reply:
I'm not sure where exactly ember does it but I'm sure it does. If I don't specify this I'm getting runtime errors like this one:

<img width="549" alt="image" src="https://github.com/embroider-build/embroider/assets/944286/12e0438d-08fb-4e30-9845-a12cacc6d46f">

Do you want me to dig for it to see where/why/how it's done?

Reply:
Maybe it's related to a fact that we're still on ember 3.x for some of the apps...